src/signal.rs

```diff
@@ -1042,6 +1139,16 @@ pub fn noise_simplex<S>(phase: Phase<S>) -> NoiseSimplex<S> {
 
 //// Trait Implementations for Signal Types.
 
+impl<F> Signal for Box<Signal<Frame=F>> 
```

Comment:
Nice, I've been meaning to add this, thanks for adding it!

I think this probably would have been best in a separate PR, but I'm happy to merge as is after the inspect change is addressed :+1: